### DISCUSSION TITLE
storybook

### DISCUSSION THREAD

**Discussion by storybook-geek** [babel/babel#11637] [Q&A] [answered]
hey,
I am starting to learn storybook and how to design addons in the react components but in the installation part only I am facing some serious issues .Please help me out in resolving those. I triggered the following command

npx -p @storybook/cli sb init --type react

and got the error (please see the snapshot)
![error](https://user-images.githubusercontent.com/66069191/83139733-7309fa00-a10a-11ea-8c3e-181f725dd42d.PNG)

**Comment by existentialism** [comment, marked as answer]
Check out https://gist.github.com/hzoo/df94af2d2ec48696e683b772c775aa67 for more info!

It's definitely published: https://www.npmjs.com/package/@babel/helper-module-transforms
